pkg/get/get.go

```diff
@@ -252,6 +278,41 @@ https://github.com/bitnami-labs/sealed-secrets/releases/download/{{.Version}}/ku
 
 {{- if and ( not ( or ( eq $arch "arm") ( eq $arch "arm64")) ) ( or ( eq .OS "darwin" ) ( eq .OS "linux" )) -}}
 https://github.com/bitnami-labs/sealed-secrets/releases/download/{{.Version}}/kubeseal-{{.OS}}-{{$arch}}
+{{- end -}}`,
+		},
+		{
+			Owner:   "inlets",
+			Repo:    "inletsctl",
+			Name:    "inletsctl",
+			Version: "0.5.4",
+			URLTemplate: `
+{{$fileName := ""}}
+{{- if eq .Arch "armv6l" -}}
+{{$fileName = "inletsctl-armhf.tgz"}}
+{{- else if eq .Arch "armv7l" }}
+{{$fileName = "inletsctl-armhf.tgz"}}
+{{- else if eq .Arch "arm64" -}}
+{{$fileName = "inletsctl-arm64.tgz"}}
+{{ else if HasPrefix .OS "ming" -}}
+{{$fileName = "inletsctl.exe.tgz"}}
+{{- else if eq .OS "linux" -}}
+{{$fileName = "inletsctl.tgz"}}
+{{- else if eq .OS "darwin" -}}
+{{$fileName = "inletsctl-darwin.tgz"}}
+{{- end -}}
+https://github.com/inlets/inletsctl/releases/download/{{.Version}}/{{$fileName}}`,
+			BinaryTemplate: `{{ if HasPrefix .OS "ming" -}}
```

Comment:
I'm not sure I understand the idea with `BinaryTemplate`? We shouldn't write a file called TOOLNAME-armhf it should always be TOOLNAME, but on Windows, sometimes it needs a .exe suffix.

Reply:
@alexellis In this case, the download url is a `tgz` file and the binary file in the `tgz` archive has different name based on arch and platform so I don't know what to pull out from the archive without using `BinaryTemplate` or a new field. 

i saw that the struct already has a field called `BinaryTemplate`, though for different purpose (github release download url with binary file, not compressed) but the name is quite close for the purpose here as well so I reused this.

Reply:
Thank you for explaining - this helps. 

So you are looking for what file to copy from the downloaded and extracted archive?
Or are you looking for the final destination name?

We probably need the following, right?

1) Download `inletsctl-darwin.tgz"`
2) Extract `inletsctl-darwin`
3) Copy `inletsctl-darwin` to destination folder as: `inletsctl`

Reply:
Im looking for what file to copy from the extracted folder.